yarn.lock

```diff
@@ -1873,7 +1965,7 @@ side-channel@^1.0.4:
     get-intrinsic "^1.0.2"
     object-inspect "^1.9.0"
 
-source-map-js@^1.0.2:
+source-map-js@^1.0.2, "source-map-js@>=0.6.2 <2.0.0":
   version "1.0.2"
```

Comment:
Why are we using multiple versions for the same package?

Reply:
Bhai, this was generated automatically while creating the react app using vite.

Reply:
@hoh-wtag AFAIK you have different version dependancies for the other packages. So, you need both versions for that.